hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -3624,6 +3625,16 @@ public OzoneDelegationTokenSecretManager getDelegationTokenMgr() {
     return ozAdmins;
   }
 
+  public boolean isAdmin(String username) throws IOException {
```

Comment:
You might want to rebase to pick up the changes from HDDS-5206, which includes a similar isAdmin change.

Reply:
Thank you @xiaoyuyao. We will be bringing in changes from master later this week. We will resolve the conflicts then.